requirements.txt

```diff
@@ -1,15 +1,15 @@
+# This file is also processed by conda-build, so keep spaces as "pkg >=1.2"
+appdirs
+dask >=0.17.0
+holoviews
+ipywidgets >=7.2
```

Comment:
Following the discussions, we can drop this requirement.

Reply:
done in https://github.com/ContinuumIO/intake/pull/211/commits/54fe34544672e8fb61e839ecdbfd73177a78c4b3